Domain Connect Spec Draft.adoc

```diff
@@ -1550,34 +1585,38 @@ consent UX should warn the user about these conflicts. For templates,
 this would be services that would be disabled. For records, this would be
 records that would be overwritten. This could be progressively disclosed.
 
-Note: When applying the same template, DNS Providers should not detect
+Note: When re-applying the same template, ideally DNS Providers would not detect
 the conflict. Instead, the first template would be removed and the new
 instance applied. For most templates this is a benign operation, unless
 the template contains variables in host names. For consideration
 of this, see the section below.
 
 === Record Types and Conflicts

```

Comment:
Reading through the changes I now notices, that the whole section is talking about deleting or adding records, while it should be rather referring to detection of situations with or without conflict.

That the conflict resolution leads to deletion of the conflicting records upon new template application is something we describe in the previous chapter, also in a bit vague way. I'll issue a proposal to rephrase the whole thing...

Reply:
Not actual anymore

Reply:
I think you re-wrote this.